dashboard/client/src/pages/job/JobDetailLayout.tsx

```diff
@@ -4,29 +4,38 @@ import {
   RiInformationLine,
   RiLineChartLine,
 } from "react-icons/ri";
+import { Outlet } from "react-router-dom";
 import { MainNavPageInfo } from "../layout/mainNavContext";
 import { SideTabLayout, SideTabRouteLink } from "../layout/SideTabLayout";
 import { useJobDetail } from "./hook/useJobDetail";
 
-export const JobDetailLayout = () => {
+export const JobPage = () => {
   const { job } = useJobDetail();
 
+  const jobId = job?.job_id ?? job?.submission_id;
```

Comment:
Are the two equivalent though? From the `JobRow` code it seems like if it uses submission ID, there is no driver (it's unclear what this means though)

Reply:
yes, either will work in the pathing of the URL. There is a case where a job was submitted but doesn't create a driver. In those cases, we still want to link to the job detail using the submission_id instead.